src/duration-picker/ForwardStartingOptions.js

```diff
@@ -42,7 +42,9 @@ export default class ForwardStartingOptions extends PureComponent {
 
     onTimeChange = e => {
         const { dateStart } = this.props;
-        const inputValue = e.target.value;
+        const inputValue = e.target.value.split(':')
```

Comment:
ok, but we are not using it anywhere and not likely to. Its not complex or complicated also. That's why i didn't think of the necessity.